flutter/test/load_android_image_list_test.dart

```diff
@@ -125,3 +148,18 @@ SentryEvent getEvent({bool symbolicated = false}) {
   );
   return SentryEvent(exception: ex);
 }
+
+class Fixture {
+  Fixture() {
+    hub = Hub(options);
+  }
+
+  final channel = MethodChannel('sentry_flutter');
+  final options = SentryFlutterOptions(dsn: fakeDsn);
+
+  late final Hub hub;
```

Comment:
```suggestion
  late final channel = MethodChannel('sentry_flutter');
  final options = SentryFlutterOptions(dsn: fakeDsn);

  final Hub hub;
```

Reply:
that would be semantically correct

Reply:
I can't make the Hub final though and the MethodChannel doesn't need to be late.

Reply:
`hub` is already final, I just removed the `late`.
`channel` does not have to be `late`, but it can, its used in `setUp` so indeed, making it `late` isn't any helpful

Reply:
The `Hub` is inited in the constructor so it has to be `late final` otherwise it would cause a compiler error.
The `Hub` can't be inited outside the constructor because it needs the SentryOptions. 
So it is correct as is :D

Reply:
all good then, my bad, thanks :D